requirements.txt

```diff
@@ -1,3 +1,4 @@
 flask
 requests
 gunicorn
+configparser
```

Comment:
I needed this in order to run the app successfully; did you not?

Reply:
no, this is included with python3. You might need to remake your virtualenv with python3, instead of the default python2.7. 
(`mkvirtualenv --python=$(which python3) required-labels` or something)

Reply:
no, this is included with python3. We might not have made your virtualenv with python3... you might need to remake it with something like: `mkvirtualenv --python=$(which python3) required-labels`